### PR TITLE
[GITHUB-8] Update VerneMQ and Ansible Versions

### PR DESCRIPTION
**Pull request by dennisconrad**
Update VerneMQ to 1.7.0

### CODE REVIEW

**Review by nathanwebsterdotme — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -9,10 +9,9 @@ python:
   - 2.7
 
 env:
-  - ANSIBLE_INSTALL_VERSION=2.2.3.0
-  - ANSIBLE_INSTALL_VERSION=2.3.3.0
-  - ANSIBLE_INSTALL_VERSION=2.4.4.0
-  - ANSIBLE_INSTALL_VERSION=2.5.0
+  - ANSIBLE_INSTALL_VERSION=2.5.9
```
Comment (lobsterdore) [inline .travis.yml]: Not quite the latest versions.

Comment (lobsterdore) [reply .travis.yml]: Should be 2.5.14 and 2.6.13.

Comment (lobsterdore) [reply .travis.yml]: Sorry :)

Comment (dennisconrad) [reply .travis.yml]: Good catch.  My script that pulls the latest versions filters on the string value, not the int value :-(

**Review by wojtek-oledzki — APPROVED** [review]

**Pull request merged**